runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingPlugin.java

```diff
@@ -335,4 +346,39 @@ public void onVarbitChanged(VarbitChanged event)
 			trawlerNotificationSent = false;
 		}
 	}
+
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() == WidgetID.FISHING_TRAWLER_GROUP_ID)
+		{
+			startTime = Instant.now();
+		}
+	}
+
+	/**
+     * Changes the Fishing Trawler timer widget from minutes to minutes and seconds
+     *
+     */
+	private void setTrawlerTimer()
+	{
+		int regionID = client.getLocalPlayer().getWorldLocation().getRegionID();
```

Comment:
you should probably add null-check for startTime at the start of the method and early return